### DISCUSSION TITLE
Do I have to remove a movie from both Plex & Trakt's watchlist to prevent it being re-added.

### DISCUSSION THREAD

**Discussion by danhew** [Taxel/PlexTraktSync#1114] [Q&A] [answered]
With both `Trakt to Plex` and `Plex to Trakt` watchlist syncing enabled, would I need to remove a movie from both watchlists to permanently remove it?

So, If I remove a movie from Trakt's watchlist and not Plex, will the movie will get re-added to Trakt and visa versa? Or does one take precedence and is the master list.

**Comment by dantebarba** [comment, marked as answer]
I recommend not to have bidirectional sync enabled: Configure the script like this and try to always use trakt watchlist

https://github.com/Taxel/PlexTraktSync/discussions/1067

**Reply by danhew** [reply]
Thanks for the tip. I'll do it that way from now on.
